examples/pytorch/hpu/mnist_sample.py

```diff
@@ -63,7 +63,7 @@ def configure_optimizers(self):
             "accelerator": "hpu",
             "devices": 1,
             "max_epochs": 1,
-            "plugins": lazy_instance(HPUPrecisionPlugin, precision="16-mixed"),
+            "plugins": lazy_instance(PrecisionHPU, precision="16-mixed"),
```

Comment:
Why are all the names backward? It should be HPUPrecision, HPUAccelerator, and so on.

Reply:
hmm I can rever it but found it strange that had `HPUSingleStrategy`
so ho I read it a posix - Accelerator for HPU

Reply:
I agree with @awaelchli . let's be consistent with the naming we have everywhere else which is a device-prefix

Reply:
ok, will revert it but still would instead do this way :D